docs/about-filecoin/primer.md

```diff
@@ -0,0 +1,147 @@
+
+Supplemental Materials:
+
+- [Filecoin: A Decentralized Storage Network](https://filecoin.io/filecoin.pdf) (2017 whitepaper)
+- [The Future of Decentralized Social Media with Balaji Srinivasan, Vitalik Buterin, Juan Benet](https://www.youtube.com/watch?v=DTxE9KV3YrE&list=PLXzKMXK2aHh43-aCLLuGnny7nhnsf49LO&index=7&t=9s)
+- [ProtoSchool: Decentralized Data Structures](https://proto.school/data-structures/)
+
+### [How Filecoin works](https://www.youtube.com/watch?v=P28aNAdZDi4)
+
+This workshop is a conceptual overview of how Filecoin works, including background on the project’s mission, a spec system overview, the key players in the network, and an overview of the Web3 ecosystem.
+
+Length: 2 hours
+
+Supplemental Materials:
+
+- Watch the [3D Filecoin Explainer](https://filecoin.io/) (click “Explore the filecoin vision”)
+- [Power Fault Tolerance - Technical Report](https://research.protocol.ai/publications/power-fault-tolerance/)
+- [Proof of Replication - Technical Report](https://filecoin.io/proof-of-replication.pdf)
+
+### [Intro to Filecoin mining](https://filecoin.io/blog/filecoin-guide-to-storage-mining/)
+
+There are two primary roles a node can play in the Filecoin network: storage and retrieval. In this session, Filecoin Software Engineer Łukasz Magiera explains how to mine on the Filecoin Network, walking through key concepts including storage sectors, storage proofs, and mining power. He offers a high-level overview of the network’s architecture and covers how to set up a mining node.
+
+Length: 1 hour
+
+Supplemental Materials:
+
+- [A Guide to Filecoin Storage Mining](https://filecoin.io/blog/filecoin-guide-to-storage-mining/)
+- [How to mine on the testnet](https://docs.filecoin.io/how-to/install-filecoin/)
+- [Troubleshooting mining issues](https://www.youtube.com/watch?v=D9qdv_3yNbM)
+- [Proof of Replication - Technical Report](https://filecoin.io/proof-of-replication.pdf)
+
+### [Economics of Filecoin](https://www.youtube.com/watch?v=fzGlen-LkKA)
+
+In this Zoom presentation, the Filecoin team discusses the mechanisms and models presented in the recent report _Engineering Filecoin’s Economy_. We will also discuss a number of questions submitted by the community. To have your questions covered, please submit your questions in the linked form below.
+
+Length: 2 hours
+
+Supplemental Materials:
+
+- [Introducing the Filecoin Economy](https://filecoin.io/blog/introducing-the-filecoin-economy/)
+- [Engineering Filecoin’s Economy](https://filecoin.io/2020-engineering-filecoins-economy-en.pdf)
+- [Filecoin’s Cryptoeconomics Constructions](https://filecoin.io/blog/filecoin-cryptoeconomic-constructions/)
+
+## The Filecoin ecosystem
+
+### [Filecoin integrations for web3 infrastructure](https://www.youtube.com/watch?v=Q0oe6i7d1u4)
+
+Filecoin presents many promising business opportunities. This session explores a few key Filecoin integration opportunities for pinning services and other businesses looking to integrate Filecoin.
```

Comment:
We have a pending guide for the "Filecoin ecosystem". It would be good to write it?